src/Hosting/Hosting/test/HostingApplicationDiagnosticsTests.cs

```diff
@@ -0,0 +1,532 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Hosting.Tests
```

Comment:
These aren't new but I renamed files (my bad).
- HostingApplicationDiagnosticsTests used to be HostingApplicationTests
- HostingApplicationTests contain the new tests

Reply:
I uses `git mv ` to ensure it git picks up file renames